DAS/pyproject.toml

```diff
@@ -5,20 +5,20 @@ description = ""
 authors = ["Blake <bhay0010@student.monash.edu>"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
-paho-mqtt = "^1.5.0"
-urllib3 = "^1.25.9"
-pandas = "^1.0.3"
```

Comment:
These have just been re-arranged because I wasn't able to simply downgrade the python version here, I had to delete `pyproject.toml` and recreate it with `poetry init`. Hence, all the libraries are re-ordered.